src/mongo/db/storage/mmap_v1/file_allocator.cpp

```diff
@@ -313,6 +313,7 @@ void FileAllocator::ensureLength(int fd, long size) {
             uassert(10443, errnoWithPrefix("FileAllocator: file write failed"), written > 0);
             left -= written;
         }
+		log() << "filling zeroes in file..." << endl;
```

Comment:
1. Please fix the leading whitespace (spaces only, 4 per indent).
2. Please move this to be immediately prior to where the zeroing starts.  Since the zeroing may take some time, it is important to provide feedback in the log prior to when it starts (ie. line 300).  There is no need to log when it finishes, since that's done anyway on line 433.